activerecord/lib/active_record/core.rb

```diff
@@ -98,6 +98,13 @@ def self.error_on_ignored_order_or_limit=(value)
         self.error_on_ignored_order = value
       end
 
+      # :singleton-method:
+      # Specify the behavior for unsafe raw query methods. A value of true will
+      # rename methods like #order to #unsafe_raw_order and will validate
+      # arguments to #order. A value of false will do nothing.
+      mattr_accessor :guard_unsafe_raw_sql, instance_writer: false
+      self.guard_unsafe_raw_sql = false
```

Comment:
I think this would be better as `allow_unsafe_raw_sql` so that it's more likely to motivate someone to use it.